src/main/java/com/faforever/client/main/event/NavigationItem.java

```diff
@@ -10,7 +10,11 @@
   LEADERBOARD("theme/leaderboard/leaderboards.fxml"),
   UNITS("theme/units.fxml"),
   TUTORIALS("theme/tutorial.fxml"),
-  TOURNAMENTS("theme/tournaments/tournaments.fxml");
+  TOURNAMENTS("theme/tournaments/tournaments.fxml"),
+
+  CUSTOM_GAMES("theme/play/custom_games.fxml"),
+  RANKED_1V1("theme/play/ranked_1v1.fxml"),
+  COOP("theme/play/coop/coop.fxml");
```

Comment:
Do u know what u are doing here?

Reply:
No :P
I'm currently trying to get my head around this eventhandler stuff. It's not supposed to stay like that.